app/assets/javascripts/school_administrator_dashboard/DashboardTestData.js

```diff
@@ -1,9 +1,16 @@
-export function testSchool() {
-  return {
-    local_id: 'HEA',
-    name: 'Arthur D. Healey'
-  };
-}
+export const testSchool = {
+  local_id: 'HEA',
+  name: 'Arthur D. Healey'
+};
```

Comment:
Can you say more about changing these from functions to shared global objects?  One benefit of functions is that there's no possibility of a test polluting the value, since you're creating a new test object each time.  If it's a global object, it opens up a class of bugs where test code or real code could update the object and break completely unrelated tests.

Reply:
Ah, you're right. I was thinking of this as static fixture data, but there's no reason to switch while it's still a shared component.